plugins/hls-splice-plugin/hls-splice-plugin.cabal

```diff
@@ -18,6 +18,7 @@ library
   build-depends:    aeson
                   , base       >=4.12 && <5
                   , containers
+                  , extra
                   , foldl
                   , haskell-lsp
                   , hls-plugin-api
```

Comment:
Alternatively: don't incur a dependency just to satisfy hlint's obsession with eitherM. But the rest of hls already depends on extra so why not?

Reply:
HLint only suggests extra functions if you pass the appropriate option, so if you don't want to use more option, then don't pass that.